src/main/java/me/zeroeightsix/kami/module/modules/render/NoRender.kt

```diff
@@ -71,4 +135,37 @@ object NoRender : Module() {
             }
         }
     }
+
+    private fun updatelist() {
+        entityList = HashSet()
+        if (mob.value) {
+            entityList.add(EntityAnimal::class.java)
+            entityList.add(EntityCreature::class.java)
+            entityList.add(EntityAmbientCreature::class.java)
+            entityList.add(EntityTameable::class.java)
+            entityList.add(AbstractHorse::class.java)
+            entityList.add(AbstractChestHorse::class.java)
+            entityList.add(EntityGolem::class.java)
+        }
+        if (player.value) entityList.add(AbstractClientPlayer::class.java)
+        if (xp.value) entityList.add(EntityXPOrb::class.java)
+        if (paint.value) entityList.add(EntityPainting::class.java)
+        if (enchantingTable.value) entityList.add(TileEntityEnchantmentTable::class.java)
+        if (sign.value) entityList.add(TileEntitySign::class.java)
+        if (skull.value) entityList.add(TileEntitySkull::class.java)
+        if (falling.value) entityList.add(EntityFallingBlock::class.java)
+        if (armorStand.value) entityList.add(EntityArmorStand::class.java)
+        if (endPortal.value) entityList.add(TileEntityEndPortal::class.java)
+        if (banner.value) entityList.add(TileEntityBanner::class.java)
+        if (itemFrame.value) entityList.add(EntityItemFrame::class.java)
+        if (items.value) entityList.add(EntityItem::class.java)
+        if (crystal.value) entityList.add(EntityEnderCrystal::class.java)
+    }
```

Comment:
Replace this with a boolean expression check, `::class.java` requires reflection and it would have an impact on performance.

Reply:
can you elaborate on that how would i get the specific lass object into the list ?

Reply:
Use the entity as a parameter and check the type with `is`

Reply:
im not sure if i understand it right. like calling a Funktion in norender.kt with the entity as a parameter that i call from the mixins and that  does an is check for every enabled option ?
this code only gets executed when you change the setting i dont see a performance problem having it this way or similar